lib/qesapdeployment.pm

```diff
@@ -255,6 +253,24 @@ sub qesap_pip_install {
     qesap_venv_cmd_exec(cmd => $pip_ints_cmd, timeout => 720);
 }
 
+
+=head3 qesap_galaxy_install
+
+  Install all Ansible requirements of the qe-sap-deployment
+=cut
+
+sub qesap_galaxy_install {
+    my %paths = qesap_get_file_paths();
+    my $galaxy_install_log = '/tmp/galaxy_install.txt';
+
+    my $ans_req = "$paths{deployment_dir}/requirements.yml";
```

Comment:
new function using `ansible-galaxy install` command and `requirements.yml` file.
The `requirements.yml` file will be added later in `qe-sap-deployment` repositiry.

The  `requirements.yml` file is what is documented https://docs.ansible.com/ansible/latest/galaxy/user_guide.html#installing-roles-and-collections-from-the-same-requirements-yml-file